### DISCUSSION TITLE
We were having some issues with an internal rate limit, which has now been sorted

### DISCUSSION THREAD

**Discussion by madpilot** [amberelectric/public-api#215] [Announcements]
We're having some issues with internal rate limiting on the public API - one of the services we use internally is hitting a limit, which is bubbling up a 429. This is not due to the API rate limits we have in place.

Looking in to it.

**Comment by madpilot** [comment]
Fixed!

**Comment by NickElseySpelloC** [comment]
I got a 429 a few days ago - is the problem back?
